test/unit/models/inbound/consentRequests.model.test.ts

```diff
@@ -51,6 +53,7 @@ describe('InboundConsentRequestsRequestModel', () => {
     beforeEach(async () => {
       thirdpartyRequests = {
         postConsents: jest.fn(() => Promise.resolve()),
+        putConsentRequestsError: jest.fn(() => Promise.resolve()),
       } as unknown as ThirdpartyRequests
 
       dfspBackendRequests = {
```

Comment:
`getScopes` is returning an Array as the root object, it is antipattern for JSON based APIs - there is a legacy security issue (any DFSP will start complaining when they have to implement it on their side):
- https://stackoverflow.com/questions/3503102/what-are-top-level-json-arrays-and-why-are-they-a-security-risk
- https://medium.com/@rm.dev/securing-json-apis-with-wrapper-objects-top-level-json-array-security-issue-1cf855468b8c

let use data property instead
```{ scopes: [....] }```

Another good point to avoid root Arrays is how easy it would be to extend response (if needed in the future): for Object root, you can simply add new optional properties without breaking the interface for old clients whereas using Array root you are locked down.